lib/src/lib.rs

```diff
@@ -7,142 +7,7 @@
 //! reloading from a file regularly. The event loop runs in its own thread
 //! and receives commands through a message queue.
 //!
-//! To create a HTTP proxy, you first need to create a `HttpProxyConfiguration`
```

Comment:
As mention globally, this should be kept, because it is displayed in the documentation.

Reply:
Okay, it's done!